EasyCommands/CommandParsers/ParameterParsers.cs

```diff
@@ -460,7 +462,7 @@ partial class Program {
             ? NewList<Token>()
             : TokenizeEnclosed(commandString, "`\'\"",
                 u => u.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries)
-                    .SelectMany(v => firstPass(v))
+                    .SelectMany(v => firstPass(v.Replace(" : ", " :: ")))
```

Comment:
Could be separated as it's own "Pass" but this would be the minimum needed

Reply:
I am stepping through the tokenizer and parser right now and this does not do anything right now, which defeats its prurpose.

Reply:
OK. Actually for obvious reasons.
The line above the changes removes all whitespace ...

Reply:
Ya, i put it in the wrong space.  Finally have some proper time to send this out with proper tests.  Sorry, been rushing things cuz i haven't had any time.

I'll send out and update, with more tests, today

Reply:
i cherrypicked this on top of the variable vector branch and will push that in a few minutes.